src/Strategy.sol

```diff
@@ -6,36 +6,95 @@ pragma experimental ABIEncoderV2;
+        IERC20(0x049d68029688eAbF473097a2fC38ef61633A3C7A);
+    IERC20 internal constant nUSD =
+        IERC20(0xED2a7edd7413021d440b09D654f3b87712abAB66);
+    IERC20 internal constant SYN =
+        IERC20(0xE55e19Fb4F2D85af758950957714292DAC1e25B2);
+
+    // lp tokens
+    IERC20 internal immutable syn3PoolLP;
+
+    // staking contracts
+    IMasterChef internal immutable synStakingMC;
+
+    // pools
+    ISwap internal immutable syn3PoolSwap; // Synapse Fantom Stable 3 Pool
+
+    // router
+    IUniswapV2Router02 internal immutable solidlyRouter;
+
+    //1	    0.01%
+    //5	    0.05%
+    //10	0.1%
+    //50	0.5%
+    //100	1%
+    //1000	10%
+    //10000	100%
+    uint256 public maxSlippageIn; // bips
+    uint256 public maxSlippageOut; // bips
+
+    uint256 internal constant ONE_HUNDRED_PERCENT = 10_000; // 100% bips
+
+    uint256 internal immutable pid; // Staking contract Pool ID
+    uint8 internal immutable syn3PoolUSDCTokenIndex; // Index of USDT in Synapse Fantom 3 Pool
+
+    constructor(
+        address _vault,
+        address _synStable3PoolLP,
+        address _synStable3Pool,
+        address _solidlyRouter,
+        address _synStakingMC,
+        uint256 _stakingContractPoolId,
+        uint8 _synStable3PoolUSDCTokenIndex,
+        uint256 _maxSlippageIn,
+        uint256 _maxSlippageOut
+    ) BaseStrategy(_vault) {
+        require(_maxSlippageIn <= ONE_HUNDRED_PERCENT, "maxSlippageIn too high");
+
+        require(
+            _maxSlippageOut <= ONE_HUNDRED_PERCENT,
+            "maxSlippageOut too high"
+        );
+        minReportDelay = 60 * 60 * 24 * 7; // 7 days
+
+        syn3PoolLP = IERC20(_synStable3PoolLP); // FTM Mainnet: 0x2DC777ff99058a12844A33D9B1AE6c8AB4701F66
+        syn3PoolSwap = ISwap(_synStable3Pool); // FTM Mainnet: 0x85662fd123280827e11C59973Ac9fcBE838dC3B4
+        synStakingMC = IMasterChef(_synStakingMC); // FTM Mainnet: 0xaeD5b25BE1c3163c907a471082640450F928DDFE
+        solidlyRouter = IUniswapV2Router02(_solidlyRouter); // FTM Mainnet: 0xa38cd27185a464914D3046f0AB9d43356B34829D
+
+        maxSlippageIn = _maxSlippageIn;
+        maxSlippageOut = _maxSlippageOut;
+
+        pid = _stakingContractPoolId; // FTM Mainnet: 3
+        syn3PoolUSDCTokenIndex = _synStable3PoolUSDCTokenIndex; // FTM Mainnet: 1
     }
 
     // ******** OVERRIDE THESE METHODS FROM BASE CONTRACT ************
 
-    function name() external view override returns (string memory) {
+    function name() external pure override returns (string memory) {
         // Add your own name here, suggestion e.g. "StrategyCreamYFI"
-        return "Strategy<ProtocolName><TokenType>";
+        return "StrategySynapseUSDC";
     }
 
     function estimatedTotalAssets() public view override returns (uint256) {
-        // TODO: Build a more accurate estimate using the value of all positions in terms of `want`
-        return want.balanceOf(address(this));
+        return scaledLPtoWant() + wantBalance();
```

Comment:
Changed this to only include LP and want balance.
LP is scaled by virtual price.